appIos/feed/feed/FeedScreen.swift

```diff
@@ -48,7 +48,7 @@ public struct FeedScreenContent: View {
         .onChange(of: searchQuery) { _ in
             viewModel.onSearchTextChanged(newQuery: searchQuery)
         }
-        .searchable(text: $searchQuery, prompt: "Search articles...")
+        .searchable(text: $searchQuery, prompt: "Search \(state?.searchQuery?.lowercased() ?? "" )")
```

Comment:
I found few limitations with this:
- Whilst the `Cancel` button shows, the query is always capitalized in the Search box, can that be changed? does it make sense for it to be lowercased all the time?
- Is this the expected place? The search bar is hidden until the user pulls majority of time, not sure if it's informative  enough

Reply:
Ignore this comment, I then saw the other item on the task list about defaulting to `brexit,fintech`

Reply:
I think having it capitalised is probably okay. 

Maybe the searchbox isn’t the best place to put this but is probably okay for v1 to illustrate the concept? We can always make it a bit better later on with some design with Figma.